apps/test/unit/p5lab/spritelab/CoreLibraryTest.js

```diff
@@ -169,6 +176,36 @@ describe('SpriteLab Core Library', () => {
     });
   });
 
+  describe('Sprite limit', () => {
+    // The number of sprites created is limited by (MAX_NUM_SPRITES + 1)
+    it('caps at (MAX_NUM_SPRITES + 1) or 1001 sprites', () => {
```

Comment:
nit: if you want to make this test future-proof in case MAX_NUM_SPRITES changes, you can convert the test name to a template string and replace the 1001 with ${MAX_NUM_SPRITES + 1} 🙂